tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -302,6 +343,16 @@ function install_artifacts() {
 					;;
 			esac	
 		fi
+
+		if [ -n "${INSTALLATION_PREFIX}" ]; then
+			# We could always do this sed, regardless, but I have a strong preference
+			# on not touching the configuration files unless extremelly needed
+			sed -i -e "s|${default_dest_dir}|${dest_dir}|g" "${kata_config_file}"
+
+			# Let's only adjust qemu_cmdline for the QEMUs that we build and ship ourselves
+			[[ "${shim}" =~ ^(qemu|qemu-snp|qemu-nvidia-gpu|qemu-nvidia-gpu-snp|qemu-sev|qemu-se)$ ]] && \
```

Comment:
Do the other hypervisors work fine without this adjustment, or is the feature only supported on qemu?

Reply:
Other hypervisor do not depend on such bios being present in the system.  So, the issue we had is QEMU specific so far, and that's the reason we're doing the change only for QEMU.